packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1388,6 +1388,7 @@
   "No_pages_yet_Try_hitting_Reload_Pages_button": "No pages yet. Try hitting \"Reload Pages\" button.",
   "No_messages_yet": "No messages yet",
   "No_pinned_messages": "No pinned messages",
+  "No_reactions": "No Reacting",
```

Comment:
Shouldn't this be "No Reactions"? 🤔

Reply:
Agreed unless I'm not understanding the context.  This should for sure be "No Reactions"